site/pages/review-disruption/[disruptionId].page.tsx

```diff
@@ -383,6 +400,22 @@ const ReviewDisruption = ({
                                                         ),
                                                     ],
                                                 },
+                                                {
+                                                    header: "HootSuite profile",
+                                                    cells: [
+                                                        post.hootsuiteProfile,
+                                                        createChangeLink(
+                                                            "hootsuite-profile",
+                                                            CREATE_SOCIAL_MEDIA_POST_PAGE_PATH,
+                                                            disruption,
+                                                            nextIndexSocialMedia,
```

Comment:
Same for all of the links linking to the existing social media posts

Reply:
done